backend/src/controllers/link.controller.js

```
@@ -0,0 +1,165 @@
+        "CREATE_LINK_ERROR",
+        err.message
+      );
+      res.status(statusCode).json(payload);
+    }
+  }
+
+  async deleteLink(req, res) {
+    try {
+      const { id } = req.params;
+
+      if (isNaN(id) || id.trim() === "") {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)